src/lm_polygraph/stat_calculators/prompts.py

```diff
@@ -0,0 +1,118 @@
+CLAIM_EXTRACTION_PROMPTS = {"en" : """Please breakdown the sentence into independent claims.
```

Comment:
Could we rename this file to `src/lm_polygraph/stat_calculators/claim_level_prompts.py`? This directory mostly contains sequence-level code, so a more specific name might improve organization.

Reply:
Good idea, better distinguishable with prompt.py as well. Renamed.